build-system/server.js

```diff
@@ -50,13 +52,23 @@ setInterval(function() {
   }
 }, 1000);
 
+const middleware = [app];
```

Comment:
I believe `app` also needs to be gated by `quiet`.

Reply:
Done, but can you explain why?

Reply:
I added the `--quiet` flag because the logs from the `gulp serve` background process were interfering with the debug logs from Percy during visual diff runs. See #10422. When I investigated the sources of those logs, they appeared to come from two places, and I had to silence both `app` and `morgan`. (Logging statements from `app` may not be all that common during `gulp serve` today, but they did exist a couple months ago.)

Reply:
Aha, I think I remembered why. IIRC, `app` was causing several `Powered by AMP ⚡` messages to be thrown in at different places in the Percy debug logs.